CMP1903_A1_2324/Program.cs

```diff
@@ -14,6 +15,32 @@ static void Main(string[] args)
              * Create a Game object and call its methods.
              * Create a Testing object to verify the output and operation of the other classes.
              */
+
+            /// <summary>
+            /// The "Program" class contains the code to be ran by the program, in this case it includes the "Testing" method and "Game" method which are both instantiated and executed,
+            /// the Testing is done first and then the game will run in a loop until the user tells it to stop.
+            /// </summary>
+
+
+            Game Game1 = new Game(); //Object instantiation, loading "Game" into the program class.
+
+            Testing Test1 = new Testing(); //Object instantiation, loading "Testing" into the program class.
```

Comment:
Adding a potential "Testing" boolean variable in order to enable or disable testing mode so it is not always running in the code.